test/bigml/test/api/source.clj

```diff
@@ -8,11 +8,13 @@
   (:use clojure.test))
 
 (defn- create-and-test [artifact & params]
-  (let [initial (api/get-final (apply source/create artifact params))
+  (let [{:keys [username api_key dev_mode] :as sticky-pars} params
```

Comment:
Minor nit - there's no need to bind `sticky-pars` here as it will be the same as `params`.  Also, neither `username`, `api_key`, or `dev_mode` are used in the `let`.  So this could be simplified to:

```clojure
(let [sticky-pars (flatten (seq params))
      ...
```